models/wrf/README.rst

```diff
@@ -0,0 +1,379 @@
+==========
+WRF README
+==========
+
+Contents
+========
+
+#. `Overview`_
+#. `Namelist`_
+#. `Files`_
+#. `Terms of Use`_
+#. `References`_
+
+Overview
+========
+
+DART interface module for the WRF model. This page documents the details of the
+module compiled into DART that interfaces with the WRF data in the state vector.
+There is additional overview and tutorial documentation for running a WRF/DART
+assimilation in ``./tutorial/README.md``.
```

Comment:
Can we please make this sentence BOLD. It's important.

Reply:
Done.